app/assets/javascripts/shared/behaviors.js

```diff
@@ -6,4 +6,27 @@ document.addEventListener('turbolinks:load', function(){
   $('[data-behavior~=rich-toolbar]').each(function() {
     new EditorToolbar($(this));
   });
-})
+
+  // Activate Modals for images within textile rendered areas
```

Comment:
20 lines of initialisation seems like enough to be moved to their own file instead of leaving them here. I think we need an easier "if this is a thing in the page, trigger the full init, which lives elsewhere" approach.